lib/pure/concurrency/cpuload.nim

```diff
@@ -11,7 +11,7 @@
 ## creating a thread is a good idea.
 
 when defined(windows):
-  import winlean, os, strutils, math
+  import winlean, os, strutils, math, random
```

Comment:
Please move this import into the ``when not defined(testing) and isMainModule:`` section.